logview/src/logview-filter.c

```diff
@@ -116,6 +116,10 @@ logview_filter_get_property (GObject *object, guint prop_id, GValue *value, GPar
     case PROP_TEXTTAG:
       g_value_set_object (value, priv->tag);
       break;
+    default:
+      /* We don't have any other property... */
```

Comment:
This comment is not useful IMO

Reply:
In order not to write the code I did Ctrl+C / Ctrl-V from https://developer.gnome.org/gobject/stable/gobject-properties.html

Reply:
Maybe, but you didn't see fit to include that comment in the other files, so why is that one special?  IMO this made some sense in the GObject example code to be a little more didactic, but I really don't think on production code it's useful (I feel like `G_OBJECT_WARN_INVALID_PROPERTY_ID` is clear enough already), and if you really want to explain, I think a more formal comment would be better (like *"Emit a runtime warning in the event this function is called with an invalid property ID, which either mean a bug in this code (missing handling of a property ID) or in the supporting library"*)

Reply:
done